src/printer/components/control.js

```diff
@@ -57,7 +57,14 @@ function initDisableSteppersBtn() {
 
 function updateButtons(state) {
   const disabled = state.flags.printing || state.flags.pausing || state.flags.paused;
-  const whitelist = ["move-step", "extrude-retract-step"];
+  const whitelist = [
+    "move-step",
```

Comment:
This have to be divided:
**printing**:
  * flowrate
  * bed
  * nozzle
  * spped
  
**paused**:
  * all from printing plus
  * move-step
  * extrude-retract-step - i hope that this is for both operations, extrude and retract

Reply:
Paused state
![Snímek obrazovky z 2022-08-12 11-21-08](https://user-images.githubusercontent.com/5078803/184336820-84326029-dca2-4885-b86d-8b9003bf8dc7.png)